### PR TITLE
Bug 1857928: switch to amq operator from cockroach for olm sanity test

### PR DESCRIPTION
**Pull request by ecordell**
we've been seeing timeouts with cockroach

### CODE REVIEW

File: test/extended/operators/olm.go [hunk 1]
```diff
@@ -152,68 +150,69 @@ var _ = g.Describe("[sig-operator] OLM should", func() {
...
 
 		// configure OperatorGroup before tests
-		configFile, err := oc.AsAdmin().Run("process").Args("--ignore-unknown-parameters=true", "-f", operatorGroup, "-p", "NAME=test-operator", fmt.Sprintf("NAMESPACE=%s", oc.Namespace()), "SOURCENAME=community-operators", "SOURCENAMESPACE=openshift-marketplace").OutputToFile("config.json")
+		configFile, err := oc.AsAdmin().Run("process").Args("--ignore-unknown-parameters=true", "-f", operatorGroup, "-p", "NAME=test-operator", fmt.Sprintf("NAMESPACE=%s", oc.Namespace())).OutputToFile("config.json")
 		o.Expect(err).NotTo(o.HaveOccurred())
 		err = oc.AsAdmin().WithoutNamespace().Run("create").Args("-f", configFile).Execute()
 		o.Expect(err).NotTo(o.HaveOccurred())
-		err = wait.Poll(10*time.Second, operatorWait, func() (bool, error) {
+
+		o.Eventually(func() string {
```
Comment (kevinrizza) [inline test/extended/operators/olm.go:175]: +1

**Comment by bparees** [comment]
/lgtm

ping me again when this is passing tests other than verify-deps and i'll manually merge.

**Comment by bparees** [comment]
verify deps is perm broken at the moment, manually merging.

**Comment by bparees** [comment]
@ecordell this is also failing in 4.5 (and probably 4.4 and earlier?)  is this fix being backported?

**Comment by bparees** [comment]
example of 4.5 failure: https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/release-openshift-origin-installer-e2e-gcp-4.5/1291277912317628416

**Pull request merged**